.github/workflows/test-go.yml

```diff
@@ -110,9 +110,13 @@ jobs:
           path: test-results/go-test
           key: go-test-reports-${{ github.run_number }}
           restore-keys: go-test-reports-
-      - name: List cached results
-        id: list-cached-results
-        run: ls -lhR test-results/go-test
+      - name: Sanitize timing files
+        id: sanitize-timing-files
+        run: |
+            TIMING_FILES='test-results/go-test/*.json'
+            jq . "$TIMING_FILES" > /dev/null 2>&1 || TIMING_FILES='test-results/default/*.json'
```

Comment:
We might be better off just pruning bad files here as well, so we potentially have _some_ timing data, rather than degrading to the default.

Reply:
I like being defensive about it in both places.